### ISSUE TITLE
Create a basic tanstack/query example with offline support

### ISSUE THREAD

**Issue by KyleAMathews** [electric-sql/electric#1479]
A basic example but this time with offline support https://tanstack.com/query/latest/docs/framework/react/guides/mutations#persisting-offline-mutations

**Comment by msfstef** [comment]
Picking this up - should this be the same example as https://github.com/electric-sql/electric/issues/1478 ? I can easily integrate this in the PR adding the example https://github.com/electric-sql/electric/pull/1568

**Comment by KyleAMathews** [comment]
Yeah, might as well have just one tanstack/query example w/ offline support for mutations. It'd be 98% redundant otherwise
